examples/decoding/plot_receptive_field.py

```diff
@@ -122,7 +130,6 @@
 ax.set(title="Mean prediction score", xlabel="Channel", ylabel="Score ($r$)")
 mne.viz.tight_layout()
 
-###############################################################################
```

Comment:
why remove this? It will fail to capture the section title

Reply:
yeah that wasn't fully intentional.. first time playing with these sphinx examples. :D  

Is there any way to render them locally so I can see what my changes look like before committing ?

Reply:
sadly the only way is to build the docs locally :-)  one way to do this (relatively) quickly is to use the `build_pattern` flag for sphinx...see: https://martinos.org/mne/dev/contributing.html#checking-and-building-documentation

Reply:
`make clean; PATTERN=plot_receptive_field.py make html_dev-pattern; make show`

Reply:
(in `mne/doc` directory)